views/index.html

```diff
@@ -4,13 +4,11 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>devLove.io</title>
```

Comment:
pull main again and add this to the main.handlebars inside of the layouts in view folder